### ISSUE TITLE
Extracting the lit value of the image

### ISSUE THREAD

**Issue by claychinasky** [kijai/ComfyUI-IC-Light#9]
Is it possible to extract the light values of the image by for example sampling the opposite way? which could give output for the map of the light data. in my mind, process is there, just reversing it, either by inversing the model or the sampling process.
which will give us more freedom to manipulate existing image's lit values and manually fix or finetune it then reuse it in latent_bg?

**Comment by kijai** [comment]
I don't know about reverse sampling such values, but we can get the difference and maybe utilize that, even a normal map can be created this way, as lllyasviel demonstrated in the original demo. I have now made nodes and added example workflow to do that:
![image](https://github.com/kijai/ComfyUI-IC-Light/assets/40791699/94c45816-21ee-4a60-bce9-071df17bc244)

**Comment by claychinasky** [comment]
that's really cool, but what I wanted is feed its original lighting as is and the image wouldn't change after the sampling. 
main purpose is inpainting and storing the original light state so it could be partially forced over inpaints, light value usually don't match up very well on inpaints.
I tried to feed the image as latent_bg, it works somewhat, but it changes the image and multiplies the lights and shadows or giving extra color so I tried to fix it by contrast/brigness and saturation nodes. it needs some magical numbers to not the change the image.
